GCMAggregatingTableViewDataSource/GCMAggregatingTableViewDataSource/GCMAggregatingTableViewDataSource.m

```diff
@@ -57,12 +57,25 @@ - (void)addChildDataSource:(id<UITableViewDataSource>)childDataSource {
   [self.childDataSources addObject:childDataSource];
 }
 
+- (void)addChildDataSource:(id<UITableViewDataSource>)childDataSource afterDataSource:(id<UITableViewDataSource>)precedingDataSource{
+  if ([self.childDataSources containsObject:precedingDataSource]) {
+    NSInteger index = [self.childDataSources indexOfObject:precedingDataSource];
```

Comment:
Minor optimization, you could do indexOfObject: first then check to see if the index is `NSNotFound`. Prevents searching the array twice.